kartothek/serialization/_arrow_compat.py

```diff
@@ -11,6 +14,30 @@ def _fix_pyarrow_0130_table(table):
     return table
 
 
+def _fix_pyarrow_0140_table(table):
```

Comment:
question here would be if we need to fix this in the write path. We should notice this once I enable all arrow versions and we'll see if we're forward compatible